runtime/onert/api/include/nnfw_experimental.h

```diff
@@ -356,6 +356,11 @@ typedef enum
   NNFW_QUANTIZE_TYPE_U8_ASYM,
   /** symmetric quantization with a scale only */
   NNFW_QUANTIZE_TYPE_I16_SYM,
+  /** weight-only int8 symmetric quantization */
+  NNFW_QUANTIZE_TYPE_WO_I8_SYM,
+  /** weight-only int16 symmetric quantization */
+  NNFW_QUANTIZE_TYPE_WO_I16_SYM,
+
```

Comment:
I would like to introduce enumerator with `WO`, which means weight-only to distinguish with whole quantization (input and output are quantized before running).